web/src/lib/components/shared-components/side-bar/side-bar.svelte

```diff
@@ -173,4 +175,16 @@
   <div class="mb-6 mt-auto">
     <StatusBox />
   </div>
+
+  {#if $page.data.user.isAdmin}
+    <!-- TODO REMOVE IN 1.88 -->
```

Comment:
We do have the server version, you could probably compare against the actual version if you wanted to.

Reply:
I think this is good for now. I will create a PR to remove this as soon as we release 1.87